test_scripts/API/IsReady/TTS_IsReady/NotUsed/ATF_TTS_IsReady_NotRespond_Split_RPC_APPLINK_25139_WRONG_LANGUAGE.lua

```diff
@@ -486,7 +486,6 @@ function Test:initHMI_onReady_TTS_IsReady(case)
 				text_field("audioPassThruDisplayText2"),
 				text_field("sliderHeader"),
 				text_field("sliderFooter"),
-				text_field("notificationText"),
```

Comment:
`navigationText` is still present here

Reply:
removed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2495/commits/d8dce6aae7c6c8672e1a1b7e05f0e9224f7ddfd0